cylc/rose/stem.py

```diff
@@ -339,7 +339,7 @@ def _ascertain_project(self, item):
             item = os.path.abspath(os.path.join(os.getcwd(), item))
 
         if project is not None:
-            print(f"[WARN] Forcing project for '{item}' to be '{project}'")
+            print(f"[WARN] Forcing project for {item!r} to be {project!r}")
```

Comment:
IMO it's clearer what `'{foo}'` means as compared to `{foo!r}` on the basis that reusing simple constructs is simpler/lower-barrier-to-entry than learning exotic syntaxes.

I don't mind much either way but just to raise the point (as this will also hit our other repos) that I don't think it's worth the time/effort of converting a larger codebase (e.g. cylc-flow) to this pattern.

Reply:
I'm also happy for this to be a global ignore then for code readability.

Reply:
I agree - I've added a tox.ini